src/OrbitGl/DataView.cpp

```diff
@@ -9,6 +9,20 @@
 #include "App.h"
 #include "OrbitBase/File.h"
 #include "OrbitBase/Logging.h"
+#include "absl/strings/str_join.h"
+#include "absl/strings/str_split.h"
```

Comment:
nit: we use `<>` include style for external to Orbit headers

Reply:
Ah, thanks, will fix right away!

Reply:
Should be fixed now.